src/main/java/ch/njol/skript/expressions/ExprArmorSlot.java

```diff
@@ -18,69 +18,86 @@
  */
 package ch.njol.skript.expressions;
 
+import java.util.Arrays;
 import java.util.Locale;
+import java.util.stream.Stream;
 
 import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
 import org.bukkit.inventory.EntityEquipment;
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.expressions.base.PropertyExpression;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.slot.EquipmentSlot;
 import ch.njol.skript.util.slot.EquipmentSlot.EquipSlot;
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Armour Slot")
-@Description("A part of a player's armour, i.e. the boots, leggings, chestplate or helmet.")
-@Examples({"set chestplate of the player to a diamond chestplate",
-		"helmet of player is neither a helmet nor air # player is wearing a block, e.g. from another plugin"})
+@Description("Equipment of living entities, i.e. the boots, leggings, chestplate or helmet.")
+@Examples({
+	"set chestplate of the player to a diamond chestplate",
+	"helmet of player is neither a helmet nor air # player is wearing a block, e.g. from another plugin"
+})
 @Since("1.0")
```

Comment:
```suggestion
@Since("1.0, INSERT VERSION (All armour)")
```

Reply:
A little nitpick, `all armor` or `All Armor` (also preferably not armour for docs searching or it can have a searching keywords (annotation) added to them)

Reply:
```suggestion
@Since("1.0, INSERT VERSION (All Armor)")
```
Should use that then

Reply:
Armor is American, Skript uses Armour unless explicitly referencing the Spigot type.

Will add keywords annotation.